docs/TESTS.md

```diff
@@ -107,10 +115,16 @@ compliance to the style guide, install
 pip install pytest-pep8
 ```
 
+or, for Python 3.X syntax compliance...
```

Comment:
I don't understand this new part.

Reply:
Perhaps this might be better worded as "or, if you're using Python 3:"?